### ISSUE TITLE
Torment the Weak

### ISSUE THREAD

**Issue by Floorboard** [dalaranwow/dalaran-wow#3542]
Torment the Weak is not working properly, making one spec completely broken for mages (TTW Fire). Many mages confirmed this, I'm waiting to hit 80 to test it out. 

The talent is as follows :
http://wotlk.openwow.com/spell=29447

"Torment the Weak
Your Frostbolt, Fireball, Frostfire Bolt, Pyroblast, Arcane Missiles, Arcane Blast, and Arcane Barrage abilities deal 4/8/12% more damage to snared or slowed targets."

Basically it doesn't apply the buff.

If anyone else can confirm that would be great.

**Comment by robinsch** [comment]
Can you find some legit source? Because bosses are immune to snare or slows.

**Comment by Floorboard** [comment]
Of course they are, but Torment the Weak counts attack speed slows as slows for its damage buff, so things like Thunder Clap and stuff. I'm still low (as in low level) but when I get there I'll provide evidence.

**Comment by JarlBallin** [comment]
I might add that Torment the Weak works as intended in Ulduar, but not in ToC for some reason. I was told by other mages that it has something to do with ulduar actually being "inside a raid", while ToC is not. Bosses are immune to direct snare or slows like written above, but slowing boss attack speed effects/debuffs are the same as snare/slow. The following slowing boss attack speed abilities should work with Torment the Weak 

Warrior: Thunderclap
Paladin: Judgements of the Just
Druid: Infected Wounds
Death Knight: Frost Fever

**Comment by hAkonf** [comment]
http://wowwiki.wikia.com/wiki/Torment_the_Weak
_"Wrath-Logo-Small Patch 3.0.8 (2009-01-20): Now works with Arcane Blast and does bonus damage against targets afflicted with any type of slow (such as the combat slow from Thunder Clap)."_

**Comment by Vaxsysl** [comment]
Attack speed counts as slow Robinsch. Blame Blizzard for confusing tooltips.
Also, poster, its not only TTW that's cramped, its Arcane as well.

Confirming, will post screens next raiding ID if necessary.

**Comment by Vaxsysl** [comment]
I might also add that this seems bugged ever since ToC release (I was not around at the time), I'm very surprised that no mage has ever reported this.

**Comment by JarlBallin** [comment]
Torment the Weak works in Ulduar apparently, but not other places

**Comment by Floorboard** [comment]
Since I am still low level, it would be much appreciated if you,  Vaxsysl, would help in proving this issue. This issue makes specs completely worthless.

**Comment by Vaxsysl** [comment]
It's easy enough to prove, just look at damage done in ToC, drops almost 10%. I came back several weeks ago, was going to test it this coming ID anyway. I'll go to Patchwerk, XT and Gormok with two friends.

**Comment by Vaxsysl** [comment]
Tested just now with a friend. I was in frost PvP spec with TTW talented and I used Arcane Missiles (its damage is always the same, so its easy to see any effects taking place). We went to first encounters in Naxx (Anub), Ulduar (XT) and ToC (Gormok). 

Everything worked just fine at Naxx and Uld, but Gormok in ToC could not be affected by attack speed slow, so I will need to check other bosses with a raiding group if the issue is the same.

I can't upload any images atm., so if any will be required as proof, I'll do it on another day.

**Comment by Melzy** [comment]
Any ETA on this? This makes Arcane and Fireball mages needlessly gimped in Trial of the Crusader.

**Comment by robinsch** [comment]
@Rushor @Lothloryen

**Comment by JarlBallin** [comment]
Any update on this? So excited :D

**Comment by JarlBallin** [comment]
Status?

**Comment by Vaxsysl** [comment]
Still bugged.

**Comment by JarlBallin** [comment]
I was asking the developers, how's the fixing/testing on PTR?

**Comment by kalvindeane** [comment]
It's not implemented on the PTR yet.

**Comment by JarlBallin** [comment]
Status?

**Comment by Rushor** [comment]
DB/Creature: ToC Bosses - Enable apply of Torment the Weak

* closes https://github.com/dalaranwow/dalaran-wow/issues/3542
* commit: 4e3fd30598dbb561bb22bd75a33c7c98629ec19c

The issue is fixed on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by JarlBallin** [comment]
@Rushor @robinsch What's the status on this? When will it be applied to Algalon? I want to know if ICC is going to have the same problem or not

**Comment by Vaxsysl** [comment]
It will probably work fine. It's only ToC that's the problem anyway.

**Comment by Floorboard** [comment]
I'm baffled as to why this still hasn't been fixed, it's been almost a year and I got back to playing recently (decently geared 80 now) and people still report it as broken.

Can't be that hard to fix ?

**Comment by Floorboard** [comment]
If I am wrong please correct me but AFAIK in ToC it is still messed up.

**Comment by plznoz** [comment]
Yes, it is afaik.

**Comment by TeridanDW** [comment]
None of these abilities actually applies a debuff on the bosses in ToC:

> Warrior: Thunderclap
> Paladin: Judgements of the Just
> Druid: Infected Wounds
> Death Knight: Frost Fever

so I can't test the talent.
Note: Tested on beasts.

**Comment by Zarjofgt** [comment]
Theres nothing wrong with the talent, but for some reason toc bosses are immune to AS slows. Therefore there is no way to activate the requirement of the talent.
